assets/src/edit-story/components/panels/panel/panel.js

```diff
@@ -43,13 +45,39 @@ function Panel({
   initialHeight = null,
   ariaLabel = null,
   ariaHidden = false,
+  persistenceKey,
 }) {
-  const [isCollapsed, setIsCollapsed] = useState(false);
-  const [expandToHeight, setExpandToHeight] = useState(initialHeight);
-  const [height, setHeight] = useState(initialHeight);
+  const persisted = useMemo(() => {
+    const stored = localStorage.getItem(
+      `${LOCAL_STORAGE_PREFIX}:${persistenceKey}`
+    );
+    return stored && JSON.parse(stored);
+  }, [persistenceKey]);
+  const [isCollapsed, setIsCollapsed] = useState(
+    persisted?.isCollapsed || false
```

Comment:
Could probably just be this?

```suggestion
    Boolean(persisted?.isCollapsed)
```

Reply:
yeaup. good call